frontend/src/components/dashboard/aliases/BlockLevelSlider.module.scss

```diff
@@ -9,17 +8,17 @@ $trackLineHeight: 4px;
   // A CSS variable to enable different values depending on the screen size:
   --thumbDiameter: 32px;
 
-  @media screen and #{$mq-md} {
-    --thumbDiameter: 24px;
-  }
-
   display: flex;
   flex-direction: column;
   align-items: center;
   touch-action: none;
   padding: 0 $spacing-md;
   width: 100%;
 
+  @media screen and #{$mq-md} {
+    --thumbDiameter: 24px;
+  }
+
```

Comment:
question (non-blocking): Do you know if this block moved for some specific reason? Did the migrator move it?